invenio_communities/communities/services/components.py

```diff
@@ -142,6 +142,7 @@ def _create_set_from_community(self, record):
 
         community_set = OAISet()
         community_set.name = community_title
+        community_set.system_created = True
```

Comment:
IMHO I think we should create the OAI set by using the service and not by persisting the object directly in the database

Reply:
Yes indeed. The service might do extra things. The service should probably get this extra flag `system_created` `True|False` and persist it.

Reply:
it was discussed with @lnielsen, we can keep the direct usage of data layer as it is in this case, since there is no need for extra things that service provides, and it can be bypassed